### ISSUE TITLE
show in the matrix what is blocked

### ISSUE THREAD

**Issue by Haikuch** [gorhill/uMatrix#661]
maybe its already possible, if not:

I would like to see in the matrix, what exactly is blocked. For example, when there are 3 servers with blocked images, I would like to see the image urls (or just names) when I hover the (red) image/server field.

ahyes, I know its possible with the logger, but of course its a little bit complicated to open the logger, choose the page and find the right line.

**Comment by gorhill** [comment]
Duplicate of #625.

**Comment by Haikuch** [comment]
ok, so #625 is another closed issue that duplicates #78 that has been rejected by you but is not really the same what I am asking here about. I don't want to know the fingerprinting way, like the one asked at that time, but I would like to know what I'm really going to unblock when I click the matrix field. There a little hover effect would be really helpful for preotection.
